receiver/awscontainerinsightreceiver/internal/cadvisor/cadvisor_linux.go

```diff
@@ -75,7 +75,7 @@ type createCadvisorManager func(*memory.InMemoryCache, sysfs.SysFs, manager.Hous
 var defaultCreateManager = func(memoryCache *memory.InMemoryCache, sysfs sysfs.SysFs, houskeepingConfig manager.HouskeepingConfig,
 	includedMetricsSet cadvisormetrics.MetricSet, collectorHTTPClient *http.Client, rawContainerCgroupPathPrefixWhiteList []string,
 	perfEventsFile string) (cadvisorManager, error) {
-	return manager.New(memoryCache, sysfs, houskeepingConfig, includedMetricsSet, collectorHTTPClient, rawContainerCgroupPathPrefixWhiteList, perfEventsFile)
+	return manager.New(memoryCache, sysfs, houskeepingConfig, includedMetricsSet, collectorHTTPClient, rawContainerCgroupPathPrefixWhiteList, perfEventsFile, 0)
```

Comment:
@dashpole is this the correct value for us?

Reply:
yes, zero would be correct

Reply:
https://github.com/google/cadvisor/blob/9fae30700d53a3adf5f6ca764cf04de0217b833b/cmd/cadvisor.go#L76